src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ServiceCategoryDTO.kt

```diff
@@ -24,7 +24,7 @@ data class ServiceCategoryDTO(
     fun from(serviceCategory: ServiceCategory): ServiceCategoryDTO {
       return ServiceCategoryDTO(
         id = serviceCategory.id,
-        name = serviceCategory.name,
+        name = serviceCategory.name.toLowerCase(),
```

Comment:
🤔 Lowercasing a name doesn't sound like our responsibility. Can we change the contract to expect "some string" instead of a precise value?

Reply:
i agree, i'm going to remove so that the contract fails